homework-submission/W1-homework/index.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+/**
+ * A function component that inserts a static item in a list
+ * @param {props} param0 The destructured properties description and deadline
+ * @return List item with the values of the properties
+ */
+const StaticTodoItem = ({ description, deadline }) => {
+  return (
+    <li>
+      <h3>{description}</h3>
+      <span>{deadline}</span>
+    </li>
+  );
+};
+
+/**
+ * A function component that inserts a dynamic item in a list
+ * @param {props} param0 The array of todoItems
+ * @return List item with the values of the todoItems of the array if todoItem is not done yet (false)
+ * @return List item with the values of the todoItems of the array if todoItem is done (true)
+ * @return The dynamic list of todoItems
```

Comment:
I will not comment on these beautiful comments _(I like them)_, but Just a one thing to mention.
Here I would use the following as a list instead of repeating `@return` three times:
```javascript
/**
 * A function component that inserts a dynamic item in a list
 * @param {props} param0 The array of todoItems
 * @return
 *  -  List item with the values of the todoItems of the array if todoItem is not done yet (false)
 *  -  List item with the values of the todoItems of the array if todoItem is done (true)
 *  -  The dynamic list of todoItems
 */
```

Reply:
Thanks... It's done